client/components/Windows/Help.vue

```diff
@@ -590,6 +590,15 @@
 				</div>
 			</div>
 
+			<div class="help-item">
+				<div class="subject">
+					<code>/kickban nick</code>
```

Comment:
except your code works differently than what you say here.
So, which one is wrong, doc or the code 😜 ?

Reply:
You're referring to the reason, right? I'll fix the docs if that's the case

Reply:
Yes, look at other commands as to how to specify optional parameters.

Also worth noting that /ban will work differently, as that one accepts a banmask, kick only works with a nick.